README.md

```diff
@@ -24,30 +24,34 @@ release of the Triton Model Analyzer is 1.24.0 and is available on
 branch
 [r23.01](https://github.com/triton-inference-server/model_analyzer/tree/r23.01).**
 
-Triton Model Analyzer is a CLI tool to help with better understanding of the
-compute and memory requirements of the
-[Triton Inference Server](https://github.com/triton-inference-server/server/) models. These
-reports will help the user better understand the trade-offs in different
-configurations and choose a configuration that maximizes the performance of
-Triton Inference Server.
+Triton Model Analyzer is a CLI tool which can help you find a more optimal configuration for single, multiple, or ensemble models running on a [Triton Inference Server](https://github.com/triton-inference-server/server/). Model Analyzer will also generate reports to help you better understand the trade-offs of the different configurations along with their compute and memory requirements.
```

Comment:
maybe mention something about hardware in here? It isn't just generic performance, but rather it will optimize the model on the given hardware.

Reply:
Updated to add hardware mention...